pybamm/expression_tree/unary_operators.py

```diff
@@ -618,6 +618,10 @@ def _evaluates_on_edges(self, dimension):
         """See :meth:`pybamm.Symbol._evaluates_on_edges()`."""
         return False
 
+    def _sympy_operator(self, child):
+        """Override :meth:`pybamm.UnaryOperator._sympy_operator`"""
+        return sympy.Integral(child, self.integration_variable)
```

Comment:
can sympy integrals take integral limits? you can get these from the geometry. can be a separate PR

Reply:
How to access model.default_geometry from here?

Reply:
Does the `integration_variable` have this information? Otherwise might be a bit tricky...

Reply:
Hmm good point, you can't access default geometry from here, so don't worry about it. Maybe instead we could print `\Omega_n` (/s/p) as the lower limit to indicate integration over the entire domain?

Reply:
`sympy.Integral(child, sympy.symbols("\Omega_n"))` will return this

<img width="528" alt="Capture" src="https://user-images.githubusercontent.com/64051212/125668174-457e597b-2472-487e-9b82-4398a72a061a.PNG">

Reply:
Ok leave it as xn for now